internal/querynode/data_sync_service.go

```diff
@@ -36,257 +36,144 @@ const (
 	dsService.mu.Lock()
 	defer dsService.mu.Unlock()
 
-	if _, ok := dsService.collectionFlowGraphs[collectionID]; !ok {
-		dsService.collectionFlowGraphs[collectionID] = make(map[Channel]*queryNodeFlowGraph)
-	}
 	for _, vChannel := range vChannels {
-		// collection flow graph doesn't need partition id
-		partitionID := UniqueID(0)
+		if _, ok := dsService.dmlFlowGraphs[vChannel]; ok {
+			log.Warn("dml flow graph has been existed", zap.Any("Channel", vChannel))
+			continue
+		}
 		newFlowGraph := newQueryNodeFlowGraph(dsService.ctx,
-			loadTypeCollection,
+			loadType,
 			collectionID,
 			partitionID,
 			dsService.streamingReplica,
 			dsService.tSafeReplica,
 			vChannel,
 			dsService.msFactory)
-		dsService.collectionFlowGraphs[collectionID][vChannel] = newFlowGraph
-		log.Debug("add collection flow graph",
+		dsService.dmlFlowGraphs[vChannel] = newFlowGraph
+		log.Debug("add DML flow graph",
 			zap.Any("collectionID", collectionID),
+			zap.Any("partitionID", partitionID),
 			zap.Any("channel", vChannel))
 	}
 }
 
-// collection flow graph
-// addCollectionFlowGraphDelta add a collection flowGraph to collectionFlowGraphs
-func (dsService *dataSyncService) addCollectionDeltaFlowGraph(collectionID UniqueID, vChannels []string) {
+// addDeltaFlowGraphs add a flowGraph to deltaFlowGraphs
+func (dsService *dataSyncService) addDeltaFlowGraphs(collectionID UniqueID, vChannels []string) {
 	dsService.mu.Lock()
 	defer dsService.mu.Unlock()
 
-	if _, ok := dsService.collectionDeltaFlowGraphs[collectionID]; !ok {
-		dsService.collectionDeltaFlowGraphs[collectionID] = make(map[Channel]*queryNodeFlowGraph)
-	}
 	for _, vChannel := range vChannels {
-		// collection flow graph doesn't need partition id
-		partitionID := UniqueID(0)
+		if _, ok := dsService.deltaFlowGraphs[vChannel]; ok {
+			log.Warn("delta flow graph has been existed", zap.Any("Channel", vChannel))
+			continue
+		}
+		// delta flow graph doesn't need partition id
 		newFlowGraph := newQueryNodeDeltaFlowGraph(dsService.ctx,
-			loadTypeCollection,
 			collectionID,
-			partitionID,
 			dsService.historicalReplica,
 			dsService.tSafeReplica,
 			vChannel,
 			dsService.msFactory)
-		dsService.collectionDeltaFlowGraphs[collectionID][vChannel] = newFlowGraph
-		log.Debug("add collection flow graph",
+		dsService.deltaFlowGraphs[vChannel] = newFlowGraph
+		log.Debug("add delta flow graph",
 			zap.Any("collectionID", collectionID),
 			zap.Any("channel", vChannel))
 	}
 }
 
-// getCollectionFlowGraphs returns the collection flowGraph by collectionID
-func (dsService *dataSyncService) getCollectionFlowGraphs(collectionID UniqueID, vChannels []string) (map[Channel]*queryNodeFlowGraph, error) {
+// getDMLFlowGraph returns the DML flowGraph by collectionID
+func (dsService *dataSyncService) getDMLFlowGraph(collectionID UniqueID, channel Channel) (*queryNodeFlowGraph, error) {
```

Comment:
If the name is not very long, do not use abbreviations

Reply:
fixed